docs/synapse/userguides/storm_ref_cmd.rstorm

```diff
@@ -1184,20 +1476,22 @@ Help for individual ``pkg.*`` commands can be displayed using:
 
   ``<command> --help``
 
-Packages typically contain extended Storm commands and Storm library code used to implement a Storm :ref:`gloss-service`.
+Packages typically contain extended Storm commands and Storm library code used to implement a Storm
```

Comment:
"Extended Storm commands" doesn't sound right

Reply:
Yeah, I think we used the term in the past to refer to "Storm commands beyond those included in the base Synapse installation" but that's a bit legacy. Removed "extended".